src/containers/category/Category.js

```diff
@@ -1,19 +1,59 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import './Category.css';
+import pathParser from '../../services/pathparser'
+import categoryId from '../../services/categoryId'
+import { fetchSkillsActionCreator } from '../../store/actions/actions';
 
+class Category extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      initialize: false,
+    }
+  }
 
+  componentDidUpdate() {
+    if (this.props.categories.status === 200 && this.props.userLocation.status === 200 && !this.state.initialize) this.initialize();
+  }
+
+  initialize = async () => {
```

Comment:
This can go in  the `componentDidMount` without being `async` and without any state control.